scripts/us_epa/air_emissions_inventory/process_test.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for EuroStat Physical Activity process script.
+"""
+
+import os
+import unittest
+import sys
+import tempfile
+# module_dir is the path to where this test is running from.
+MODULE_DIR = os.path.dirname(__file__)
+sys.path.insert(0, MODULE_DIR)
+# pylint: disable=wrong-import-position
+from process import USCountyAirPollutionEmissionTrends
+# pylint: enable=wrong-import-position
+
+TEST_DATASET_DIR = os.path.join(MODULE_DIR, "test_data", "datasets")
```

Comment:
Rename folders
test_data = test_files
datasets = input_files

Reply:
Done the changes.